README.rst

```diff
@@ -60,6 +90,26 @@ The output of this should be roughly something like the following::
 Now loading http://localhost:8000/ should show the running site with
 the default content.
 
+The website will not have a css file yet yet. To generate the CSS files, you must first install sass using homebrew
+
+.. code-block:: bash
+
+    $ brew install sass
+
+Navigate to the repo root and run:
+
+.. code-block:: bash
+
+    $ sass unicorecmsa4w/static/sass/style.scss:unicorecmsa4w/static/css/style.css --style compressed
+    $ sass unicorecmsa4w/static/sass/enhanced.scss:unicorecmsa4w/static/css/enhanced.css --style compressed
+
+To run the pyramid application that will monitor changes in the sass files and recompile, allowing continuous changes, first ensure that you have followed the installation instructions above. Navigate to the repo root, make sure you have no other processes running in the background and then run:
+
+.. code-block:: bash
+
+    $ sh start.sh
```

Comment:
Maybe link to the `start.sh` file and say more clearly what it does (run pyramid server and sasquatch)?